recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans import AutoToolsBuildEnvironment
```

Comment:
```suggestion
from conans import AutoToolsBuildEnvironment
from conans.errors import ConanInvalidConfiguration
```
this should fix an error on CI

Reply:
Thanks, I'll try it again.